cmd/admin-policy-attach.go

```diff
@@ -74,57 +72,53 @@ func userAttachOrDetachPolicy(ctx *cli.Context, attach bool) error {
 	if len(ctx.Args()) < 2 {
 		showCommandHelpAndExit(ctx, 1) // last argument is exit code
 	}
-
-	console.SetColor("PolicyMessage", color.New(color.FgGreen))
-	console.SetColor("Policy", color.New(color.FgBlue))
+	user := ctx.String("user")
+	group := ctx.String("group")
 
 	// Get the alias parameter from cli
 	args := ctx.Args()
 	aliasedURL := args.Get(0)
 
-	// Put args in PolicyAssociationReq, client checks for validity
+	policies := args[1:]
 	req := madmin.PolicyAssociationReq{
-		User:     ctx.String("user"),
-		Group:    ctx.String("group"),
-		Policies: args.Tail(),
+		User:     user,
+		Group:    group,
+		Policies: policies,
 	}
 
 	// Create a new MinIO Admin Client
 	client, err := newAdminClient(aliasedURL)
 	fatalIf(err, "Unable to initialize admin connection.")
 
 	var e error
+	var res madmin.PolicyAssociationResp
 	if attach {
-		e = client.AttachPolicy(globalContext, req)
+		res, e = client.AttachPolicy(globalContext, req)
 	} else {
-		e = client.DetachPolicy(globalContext, req)
-	}
-
-	isGroup := false
-	if req.User == "" {
-		isGroup = true
-	}
-
-	userOrGroup := req.User
-	if isGroup {
-		userOrGroup = req.Group
+		res, e = client.DetachPolicy(globalContext, req)
 	}
+	fatalIf(probe.NewError(e), "Unable to make user/group policy association")
 
-	if e == nil {
-		for _, policy := range req.Policies {
-			printMsg(userPolicyMessage{
-				op:          ctx.Command.Name,
-				Policy:      policy,
-				UserOrGroup: userOrGroup,
-				IsGroup:     isGroup,
-			})
-		}
-	} else {
+	var emptyResp madmin.PolicyAssociationResp
+	if res.UpdatedAt == emptyResp.UpdatedAt {
+		// Older minio does not send a result, so we populate res manually to
```

Comment:
We can keep for few months, you can a comment add 'Deprecated June 2023' so it will easy to clean up later. LGTM otherwise

Reply:
Sounds good, done.